#
# This files holds the parameters for the dynamic_model, planner, and octomap
#

fuzzy_constant_speed: 1 # m/s
max_angular_vel: .5     # rad/s
timestep: .01
debug_level: 0
planner_resolution: .01
state_checker_resolution: .01   # 1%
propagation_step_size: .1
visualize_planner: 1
goal_bias: .5
distance_weights: [1,1,.1, .1,.1,.1]   